tools/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+set -e
+npm run docs:build
+cd docs/.vuepress/dist
+
+git init
+git add -A
+git commit -m 'deploy'
+git push -f git@github.com:vuejs/eslint-plugin-vue.git master:gh-pages
+
```

Comment:
What do you think about using something like this instead:

https://github.com/eldarlabs/ghpages-deploy-script/blob/master/scripts/deploy-ghpages.sh

Reply:
What are the benefits of using it instead? I copied deploy script from the official Vuepress documentation :)

Reply:
instead of making one commit and force pushing it over and over again, its adding changes only